Makefile

```diff
@@ -91,15 +91,15 @@ e2etests: ## Run the e2e suite against your local cluster
 		--ginkgo.grace-period=3m \
 		--ginkgo.vv
 
-upstream-e2etests: 
+upstream-e2etests:
 	CLUSTER_NAME=${CLUSTER_NAME} envsubst < $(shell pwd)/test/pkg/environment/aws/default_ec2nodeclass.yaml > ${TMPFILE}
 	go test \
 		-count 1 \
-		-timeout 1h \
+		-timeout 3.25h \
 		-v \
-		$(KARPENTER_CORE_DIR)/test/suites/$(shell echo $(TEST_SUITE) | tr A-Z a-z)/... \
-		--ginkgo.focus="${FOCUS}" \
-		--ginkgo.timeout=1h \
+		$(KARPENTER_CORE_DIR)/test/suites/... \
```

Comment:
Why change this? It's nice to have both directory scoping and FOCUS

Reply:
Right now, we are not scoping the tests by directory, but rather by the labels